pennylane/_qubit_device.py

```diff
@@ -183,6 +183,23 @@ def execute(self, circuit, **kwargs):
 
         return self._asarray(results)
 
+    def batch_execute(self, circuits, **kwargs):
+        """Execute a batch of quantum circuits on the device.
+
+        For plugin developers: This function is to be overwritten if the device can efficiently run multiple
+        circuits on a backend. By default, the circuits are sent to the backend one-by-one.
+
+        Args:
+            circuits (list[~.CircuitGraph]): circuits to execute on the device
```

Comment:
Would we update the terminology to `tape` later on?

Reply:
Yes, at the moment it could be a tape or a circuit graph, if I am not mistaken. I assume when tape becomes the new core we will update this.

Reply:
Yep, at the moment they both sound the same from a duck typing perspective.